modular_skyrat/modules/liquids/code/mop.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/mop/proc/attack_on_liquids_turf(obj/item/mop/the_mop, turf/tile, mob/user, obj/effect/abstract/liquid_turf/liquids)
```

Comment:
If you could do some documentation for this proc, just to explain what is does and what the arguments are there for, it'd be great, and the PR would be good to merge.